app/views/broken_window/metrics/show.html.erb

```diff
@@ -12,6 +12,16 @@
     <div class="required">
       <div class="minimum"><%= @metric.threshold_type %> <%= BrokenWindow::Metric.format_value @metric.threshold, @metric.value_type %></div>
     </div>
+    <%= tag :div, class: "chart", data: {
+      sparkline: @metric.measurements.map{ |measurement| {
```

Comment:
@measurements = @metric.measurements.order(id: :asc)

in the controller and then use it here to avoid adding default order